### PR TITLE
WIP: Reduced ping interval and disabled device resetting

### PR DESCRIPTION
**Pull request by assada**
https://github.com/PixelyIon/DeckTX/issues/1

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -139,7 +139,7 @@ class ElrsManager {
             try {
                 if (pingsSinceLastRx >= 2) {
                     fmt::print("No packets received after two pings, resetting device\n");
-                    crsfInterface.ResetDevice();
+                    //crsfInterface.ResetDevice();
```
Comment (PixelyIon) [inline src/main.cpp]: We want to retain resetting the device after enough pings, just increase the amount of pings after which we reset the device here instead. Perhaps something like 20, that would be 6 seconds?

**Review by PixelyIon — CHANGES_REQUESTED** [review]

File: src/main.cpp (hunk 1 above)
Comment (assada) [reply src/main.cpp]: It seems that the `ResetDevice` in usb case do nothing. Put it back as it was. 
I also tried keeping the PingInterval at 1 second but increased the number of pings. This didn't help. It appears that the "ping speed" is much more important than the number of pings.

I'll take a break from researching this for now because all I'm doing is monkey patching, and it's starting to annoy me a bit. =)

**Comment by assada** [comment]
Most likely, the overall idea of a direct USB connection is bad, and I'll abandon it. I'll go with making a JRBay-to-USB adapter using FTDI, so that any TX can be connected without additional configuration.

File: src/crsf/crsf.cpp [hunk 2]
```diff
@@ -4,6 +4,7 @@
 #include "protocol.h"
 #include <chrono>
 #include <thread>
+#include <algorithm>
```
Comment (PixelyIon) [inline src/crsf/crsf.cpp:7]: Are these changes adding headers + removing `constexpr` necessary for compilation on your machine? If they are then they should be put into a separate commit. Something like `treewide: Fix compilation on <compiler>`.

**Comment by PixelyIon** [comment]
> Most likely, the overall idea of a direct USB connection is bad, and I'll abandon it. I'll go with making a JRBay-to-USB adapter using FTDI, so that any TX can be connected without additional configuration.

I think that this issue will likely occur with autobaud regardless of the connection type, I'll just make this change to lower the ping interval on my repository's `main` branch directly, since this PR is closed.

File: src/crsf/crsf.cpp (hunk 2 above)
Comment (assada) [reply src/crsf/crsf.cpp:7]: Yes, it's just for me. I didn't plan to include them as part of the PR.
